control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -989,22 +988,32 @@ func createOauthService(client client.Client, hcp *hyperv1.HostedControlPlane, n
 }
 
 func ensureVPNSCC(c client.Client, hcp *hyperv1.HostedControlPlane, namespace string) error {
```

Comment:
this diff seems wrong since this changes are in the main branch already. Can you may be rebase?

Reply:
yep, I'm not even sure how this snuck into my PR